client/me/purchases/purchases-list/index.jsx

```diff
@@ -50,26 +50,7 @@ class PurchasesList extends Component {
 		if ( this.props.hasLoadedUserPurchasesFromServer && this.props.purchases.length ) {
 			content = (
 				<div>
-					{ this.props.isBusinessPlanUser && (
-						<ActionCard
-							headerText={ this.props.translate( 'Looking for Expert Help?' ) }
-							mainText={ this.props.translate(
-								'Get 30 minutes dedicated to the success of your site. Schedule your free 1-1 concierge session with a Happiness Engineer!'
-							) }
-							buttonText="Schedule Now"
-							buttonIcon={ null }
-							buttonPrimary={ true }
-							buttonHref="/me/concierge"
-							buttonTarget={ null }
-							buttonOnClick={ () => {
-								this.props.recordTracksEvent( 'calypso_purchases_concierge_banner_click', {
-									referer: '/me/purchases',
-								} );
-							} }
-							compact={ false }
-							illustration="/calypso/images/illustrations/illustration-start.svg"
-						/>
-					) }
+					{ this.props.isBusinessPlanUser && <ConciergeBanner /> }
```

Comment:
given the `<TrackComponentView />` component could we get by without separating out the `<ConciergeBanner />` into its own component and just add the tracker in-place?

```js
{ this.props.isBusinessPlanUser && (
	<>
		<TrackComponentView eventName="calypso_purchases_concierge_banner_view" … />
		<ActionCard … />
	</>
) }
```

I think the diff would end up as four lines

Reply:
I decided to implement this component but to keep the banner as a separate component so that we can reuse it.